packages/components/src/components/popup/popup-header.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon';
+import Text from '../text';
+
+const Header = ({ balance, header, header_icon, text_color, title }) => {
+    return header ? (
+        header()
```

Comment:
@aaron-binary I added this because the cashier popup had an extra button, and wanted the component to be more dynamic, but now I added that button and remove it.